code/parse/parselo.h

```diff
@@ -32,7 +32,6 @@ extern int Token_found_flag;
 
 
 #define	COMMENT_CHAR	(char)';'
-#define	EOF_CHAR			(char)-128
```

Comment:
Why not just change the define and leave the rest of the code as-is, just in case?

Reply:
That would probably break the way the parse system uses `strchr`. For example, in Line 314 in `parselo.cpp` only changing this define would write two null characters into the terminators string. I don't think it would break how the parsing works but it would be a bit inefficient.